docs-2.0/3.ngql-guide/1.nGQL-overview.md

```diff
@@ -1,18 +1,16 @@
 # Nebula Graph Query Language (nGQL)
 
-This document gives an introduction to the query language of Nebula Graph, nGQL.
-
 ## What is nGQL
```

Comment:
Why remove the preceding sentence? There suppose to be content between two titles.